rpc/server.py

```diff
@@ -142,13 +142,24 @@ def fund_account(account: string, balance: float) -> dict:
 
         # Optionally log the account creation in the transactions table
         cursor.execute(
-            "INSERT INTO transactions (from_address, to_address, data, value, type) VALUES (NULL, %s, %s, %s, 0);",
+            "INSERT INTO transactions (from_address, to_address, data, value, type) VALUES (NULL, %s, %s, %s, 0) RETURNING id;",
             (
                 current_account,
                 json.dumps({"action": "create_account", "initial_balance": balance}),
                 balance,
             ),
         )
+        new_transaction_id = cursor.fetchone()
+        cursor.execute("SELECT * FROM transactions WHERE id = %s;", (new_transaction_id,))
```

Comment:
@sevenearths do we need to fetch the transaction again? couldn't we trust that if we get the ID, then the transaction has been stored?